scene/resources/texture.cpp

```diff
@@ -406,6 +416,7 @@ void PortableCompressedTexture2D::_set_data(const Vector<uint8_t> &p_data) {
 	}
 
 	image_stored = true;
+	size_override = size;
```

Comment:
Right handling of `size_override`. Because this is produce an error if it uninitialized (`Vector2i.Zero`)